vcd/resource_vcd_catalog_vapp_template.go

```diff
@@ -47,6 +49,38 @@ func resourceVcdCatalogVappTemplate() *schema.Resource {
 				Required:    true,
 				Description: "vApp Template name",
 			},
+			"capture_vapp": {
+				Optional:    true,
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Description: "Provides configuration options for creating a vApp Template from existing vApp",
```

Comment:
I was going to suggest
```
  ConflictsWith: []string{"ova_path", "ovf_url"}
```
but I noticed that these arguments don't have it neither. I guess it's because the `switch` does the job at runtime (a bit silently I guess 😄 ), but adding this comment just in case something pops.

Reply:
I think this is a good point. I will check and most probably add it if nothing breaks.

Reply:
Changed